.github/workflows/check-publish.yml

```diff
@@ -4,9 +4,9 @@ permissions:
   contents: read
 on:
   push:
-    branches: [ main, release/* ]
+    branches: [main, release/*]
```

Comment:
I see we have different formattings for `*.yml`.
I know that @0xNeshi also prefers to format like it was before

Reply:
I'm fine either way!
Let me know which formatter you're using for _.yml_ files, and I'll install it locally

Reply:
same here!

Reply:
I know this was discussed in the past. But I think we should actually commit our `.pre-commit-config.yaml` to make things more consistent for everyone